src/main/java/build/buildfarm/worker/Executor.java

```diff
@@ -321,11 +334,14 @@ private long executePolled(
         }
       }
     }
-    return stopwatch.elapsed(MICROSECONDS) - executeUSecs;
+    return stopwatchToMicroseconds(stopwatch) - executeUSecs;
   }
 
   public void run(ResourceLimits limits) {
+    @SuppressWarnings("units")
+    @s(Prefix.micro)
     long stallUSecs = 0;
```

Comment:
Assigning constants needed a warning suppression, including in the Checker Framework examples. https://github.com/typetools/checker-framework/blob/master/docs/examples/units-extension/UnitsExtensionDemo.java